src/DependencyInjection/Configuration.php

```diff
@@ -157,6 +157,14 @@ public function getConfigTreeBuilder()
                                 'skin-yellow-light',
                             ])
                         ->end()
+                        ->enumNode('button_mode')
+                            ->defaultValue('both')
+                            ->values([
+                                'both',
```

Comment:
I think we should use a more representative term here, while bringing more flexibility to this feature.
"both" is useful only in case where 2 options are allowed, but what if we decide to add more in the future?

Reply:
What could we add in the future ? Prior to this change this was always button + text, I don't know what could be added.

Do you have another suggestion ?
An array with the value ['icon', 'text'] ? An option `all` ?

Reply:
I think "all", "full" or something like that could be good choices.
BTW, maybe we could also use other name for the option, as the current one is not clear enough to me regarding its presentation purposes. Personally, the "mode" term makes me think about other options like "readonly=yes/no" or "action=edit/show".
I guess choices like "button_view", "button_view_mode", "button_content", "button_render" could work.

Reply:
We could also add context to the list.
`action_buttons` or `list_action_buttons` seems better than `button`.